data/users.ts

```diff
@@ -0,0 +1,36 @@
+import { type UserData } from "../src/app/types/typescript";
+
+const users: UserData[] = [
```

Comment:
why is all of this data on the FE? It should be in the database?

Reply:
that file i used for the previous task 31  to export data to the database. i am fetching data from database